config.go

```diff
@@ -249,6 +250,8 @@ func xmlToFileLogWriter(filename string, props []xmlProperty, enabled bool) (*Fi
 			daily = strings.Trim(prop.Value, " \r\n") != "false"
 		case "rotate":
 			rotate = strings.Trim(prop.Value, " \r\n") != "false"
+		case "integersuffix":
+			integerSuffix = strings.Trim(prop.Value, " \r\n") != "false"
```

Comment:
I looked at configuring the agent to default to integer-based suffixes (while keeping log4go at date-based), but it looks like too much work because log4go doesn't provide an easy way to discover the property set in the XML. I'll just expose the option to use date-based suffixes in the agent and then submit a JIRA to switch the default in the next major release.